e2e/fixtures.ts

```diff
@@ -0,0 +1,20 @@
+import {Page} from '@playwright/test'
+
+export enum Editor {
+  schema = 0,
+  value = 1,
+}
+
+export const writeInMonaco = async (
+  page: Page,
+  text: string,
+  editor: Editor = Editor.value,
+  deletePreviousContent: boolean = true,
```

Comment:
I would make it an object

Reply:
agree, was too lazy 😂

Reply:
updated and now I've also left some jsdoc to document the function better

![Screenshot 2024-08-09 012418](https://github.com/user-attachments/assets/43ccaf09-bda9-474d-ac35-849acad38c41)